pkgs/development/lua-modules/generated-packages.nix

```diff
@@ -160,25 +160,27 @@ busted = buildLuarocksPackage {
 
 cassowary = buildLuarocksPackage {
   pname = "cassowary";
-  version = "2.3.1-1";
+  version = "2.3.1-2";
   knownRockspec = (fetchurl {
-    url    = "https://luarocks.org/cassowary-2.3.1-1.rockspec";
-    sha256 = "1rgs0rmlmhghml0gi4dn0rg2iq7rqnn8w8dcy9r3qsbkpyylbajc";
+    url    = "https://luarocks.org/cassowary-2.3.1-2.rockspec";
+    sha256 = "04y882f9ai1jhk0zwla2g0fvl56a75rwnxhsl9r3m0qa5i0ia1i5";
   }).outPath;
   src = fetchgit ( removeAttrs (builtins.fromJSON ''{
   "url": "https://github.com/sile-typesetter/cassowary.lua",
   "rev": "c022a120dee86979d18e4c4613e55e721c632d80",
   "date": "2021-07-19T14:37:34+03:00",
   "path": "/nix/store/rzsbr6gqg8vhchl24ma3p1h4slhk0xp7-cassowary.lua",
   "sha256": "1r668qcvd2a1rx17xp7ajp5wjhyvh2fwn0c60xmw0mnarjb5w1pq",
-  "fetchSubmodules": true,
+  "fetchSubmodules": false,
   "deepClone": false,
   "leaveDotGit": false
 }
  '') ["date" "path"]) ;
 
   disabled = (luaOlder "5.1");
   propagatedBuildInputs = [ lua penlight ];
+  checkInputs = [ busted busted ];
```

Comment:
Why are you using two `busted` `checkInputs`? I'm aware it doesn't matter that much, if `doCheck = false;` but I'm intrigued.

Reply:
Because I copy and pasted somebody else's (@teto) code assuming they *must* have had a good reason. The origin of the blooper seems to be 15e5c924959.

Reply:
I dropped it from the relevant lua-cassowary commit and also added a commit to fix it in lua-penlight. If @teto had a reason for that I guess they should be the source of the explanation you need.

Reply:
no reason especially. I wanted to enable tests in our luarocks packages but could never complete the work (anyone welcome to try :) ). Safest is to set `doCheck = false;` :'(